packages/meditrak-server/src/routes/DeleteHandler/DeleteHandler.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import {
+  constructRecordExistsWithId,
+  DatabaseError,
+  respond,
+  singularise,
+  ValidationError,
+} from '@tupaia/utils';
+import { CRUDHandler } from '../CRUDHandler';
+
+/**
+ * Responds to DELETE requests for a resource
+ */
+
+export class DeleteHandler extends CRUDHandler {
+  constructor(req, res) {
+    super(req, res);
+    this.model = this.models[singularise(this.resource)];
```

Comment:
Might be useful in other CRUDHandlers?

Reply:
Moved it up a layer into the CRUDHandler